### PR TITLE
Add First DataBase

### PR DESCRIPTION
**Pull request by Blackolf931**

### CODE REVIEW

File: DataBaseFirst/ClassLibrary1/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
```
Comment (nantonov) [inline DataBaseFirst/ClassLibrary1/Class1.cs]: Del

File: DataBaseFirst/ConsoleApp1/App.config [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
```
Comment (nantonov) [inline DataBaseFirst/ConsoleApp1/App.config]: wtf

File: DataBaseFirst/ConsoleApp1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    class Program
+    {
+        static void Main(string[] args)
```
Comment (nantonov) [inline DataBaseFirst/ConsoleApp1/Program.cs]: wtf

**Review by nantonov — CHANGES_REQUESTED** [review]

File: DataBaseFirst/ClassLibrary1/Class1.cs (hunk 1 above)
Comment (Blackolf931) [reply DataBaseFirst/ClassLibrary1/Class1.cs]: deleted

File: DataBaseFirst/ConsoleApp1/App.config (hunk 2 above)
Comment (Blackolf931) [reply DataBaseFirst/ConsoleApp1/App.config]: deleted

File: DataBaseFirst/ConsoleApp1/Program.cs (hunk 3 above)
Comment (Blackolf931) [reply DataBaseFirst/ConsoleApp1/Program.cs]: deleted

File: DataBaseFirst/DataBaseFirst/Models/Product.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+
+#nullable disable
+
+namespace DataBaseFirst.Models
+{
+    public partial class Product
```
Comment (nantonov) [inline DataBaseFirst/DataBaseFirst/Models/Product.cs:8]: Why partial?

File: DataBaseFirst/DataBaseFirst/Models/ShopContext.cs [hunk 5]
```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+#nullable disable
+
+namespace DataBaseFirst.Models
+{
+    public partial class ShopContext : DbContext
```
Comment (nantonov) [inline DataBaseFirst/DataBaseFirst/Models/ShopContext.cs:9]: Why partial?

**Review by nantonov — APPROVED** [review]

**Pull request merged**